easybuild/easyconfigs/b/Boost/Boost-1.61.0-foss-2016b-Python-2.7.12.eb

```diff
@@ -0,0 +1,26 @@
+name = 'Boost'
+version = '1.61.0'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'http://www.boost.org/'
+description = "Boost provides free peer-reviewed portable C++ source libraries."
+
+toolchain = {'version': '2016b', 'name': 'foss'}
+toolchainopts = {'usempi': True, 'pic': True}
+
+sources = ['%(namelower)s_1_61_0.tar.gz']
+source_urls = [SOURCEFORGE_SOURCE]
+
+dependencies = [
+    ('bzip2', '1.0.6'),
+    ('zlib', '1.2.8'),
+    ('Python', '2.7.12'),
+]
+
+parallel = 4
+
+start_dir = '%(builddir)s/boost_1_61_0/'
```

Comment:
this can probably be dropped too, it snuck in here because you used the easyconfig that is dumped in the easyconfig archive, which has resolved templates & the likes in it currently when using `--minimal-toolchains` (cfr. https://lists.ugent.be/wws/arc/easybuild/2016-11/msg00062.html & follow-up)